w08h03/test/pgdp/datastructures/UnitTest.java

```diff
@@ -29,13 +29,12 @@ void dynamicTests(String type, int[] expected, int[] input) {
 			tree.insert(i);
 		}
 
-		int j = 1;
+		int j = 0;
 		for (int i : tree) {
-			assertEquals(j++, i, type + ": Invalid Output at position [" + (j - 1) + "]: Expected [" + j + "], got [" + i + "]");
+			assertEquals(expected[j++], i, type + ": Invalid Output at position [" + (j-1) + "]: Expected [" + expected[j] + "], got [" + i + "]");
```

Comment:
Muss es hier nicht j - 1 sein, weil j hier schon um 1 inkrementiert worden ist (Westermann ist traurig :()? Auf jeden Fall muss j in der Position Klammer und in der Expected Klammer einheitlich sein.

Reply:
Ja…. Hab nach deinem review schnell was ungetest per Handy verändert, sollte jetzt alles passen. 👍